sites/bizbash/config/site.js

```diff
@@ -0,0 +1,85 @@
+const ads = require('./ads');
+
+module.exports = {
+  ads,
+  logos: {
+    navbar: {
+      src: 'https://base.imgix.net/files/base/bizbash/bzb/image/static/logo/site_logo.png',
```

Comment:
These needs heights specified for both regular and 2x. See another site's config for details.